packages/react-composites/src/composites/CallComposite/adapter/AzureCommunicationCallAdapter.ts

```diff
@@ -1158,6 +1169,14 @@ export class AzureCommunicationCallAdapter<AgentType extends CallAgent | BetaTea
     this.emitter.emit('transferRequested', newArgs);
   }
 
+  /* @conditional-compile-remove(capabilities) */
+  private capabilitiesChanged(data: CapabilitiesChangeInfo): void {
+    if (!data.newValue.turnVideoOn?.isPresent) {
```

Comment:
is newValue ever undefined?

Reply:
newValue is a Record and can't be undefined but can be empty. But I should be checking that turnVideoOn present in the Record explicitly with an isPresent value of `false`. I should not coercing a Boolean. Fixed.